.ci/validation.sh

```diff
@@ -88,7 +88,8 @@ eclipse-static-analysis)
   ;;
 
 nondex)
-  mvn -e --fail-never clean nondex:nondex -DargLine='-Xms1024m -Xmx2048m'
+  mvn -e --fail-never clean nondex:nondex -DargLine='-Xms1024m -Xmx2048m' \
+    -Dtest=!JavadocPropertiesGeneratorTest#testNonExistentArgument
```

Comment:
done to resolve ` JavadocPropertiesGeneratorTest.testNonExistentArgument:111 Unexpected error log ==> expected: <Missing required options and parameters: '--destfile=<outputFile>', '<inputFile>'08:59
Usage: java com.puppycrawl.tools.checkstyle.JavadocPropertiesGenerator [-hV]08:59
       --destfile=<outputFile> <inputFile>08:59
      <inputFile>   The input file.08:59
      --destfile=<outputFile>08:59
                    The output file.08:59
  -h, --help        Show this help message and exit.08:59
  -V, --version     Print version information and exit.08:59
> but was: <Missing required parameter: '<inputFile>'08:59
Usage: java com.puppycrawl.tools.checkstyle.JavadocPropertiesGenerator [-hV]08:59
       --destfile=<outputFile> <inputFile>08:59
      <inputFile>   The input file.08:59
      --destfile=<outputFile>08:59
                    The output file.08:59
  -h, --help        Show this help message and exit.08:59
  -V, --version     Print version information and exit.08:59
>`

Reply:
This error was likely caused by the picocli library; this is probably why the nondex report failed to generate a file for this.

Reply:
Please add this reason as a comment to the script file.

Reply:
Done.